pom.xml

```diff
@@ -325,6 +325,9 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>0.8.5</version>
+                <configuration>
+                    <excludes>GenerationKt</excludes>
```

Comment:
Turns out I was wrong and it doesn't work like this. Could you investigate further? Maybe, `GenerationKt.class`?